parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestParquetFileWriter.java

```diff
@@ -438,6 +443,9 @@ public void testConvertToThriftStatistics() throws Exception {
 
   @Test
   public void testWriteReadStatistics() throws Exception {
+    // this test assumes statistics will be read
+    Assume.assumeTrue(!shouldIgnoreStatistics(Version.FULL_VERSION, BINARY));
```

Comment:
Not sure I'm a fan of Assume, this could easily disable this test by accident right?

why do we need this guard?